provider/dms/Domain/Entities/Comment.php

```diff
@@ -44,13 +34,13 @@ class Comment extends Entity
      * @param string $body
      * @param IClock $clock
      */
-    public function __construct(Post $post, string $body, IClock $clock)
+    public function __construct(Post $post, string $body)
     {
         parent::__construct();
         $this->post      = $post;
         $this->body      = $body;
-        $this->createdAt = new DateTime($clock->utcNow());
```

Comment:
For the audience if we were using `datetime` field on the db, we could have used this. But as it is currently `bigint` , I am using time() ;) .

Reply:
Yes, I used int to simplify tests and avoid use yet another object to fill this field.